packages/next/client/image.tsx

```diff
@@ -405,6 +406,19 @@ function akamaiLoader({ root, src, width }: LoaderProps): string {
   return `${root}${normalizeSrc(src)}?imwidth=${width}`
 }
 
+function imageengineLoader({ root, src, width, quality }: LoaderProps): string {
+  const params = ['w_' + width]
+  let paramsString = ''
+  if (quality) {
+    params.push('cmpr_' + (100 - quality))
+  }
+
+  if (params.length) {
+    paramsString = '?imgeng=/' + params.join('/')
+  }
+  return `${root}${normalizeSrc(src)}${paramsString}`
+}
```

Comment:
Does image engine not need a `format=auto` query string or similar to switch between formats?

Reply:
No, the image format is normally dynamic and parameters are required to avoid this behavior.